app/messaging/process-message.js

```diff
@@ -0,0 +1,41 @@
+const util = require('util')
+const { set } = require('../repositories/application-repository')
+const { applicationResponseMsgType, applicationResponseQueue } = require('../config')
+const sendMessage = require('../messaging/send-message')
+const createReference = require('../lib/create-reference')
+const { notify: { templateIdApplicationComplete } } = require('../config')
+const sendEmail = require('../lib/send-email')
+const boom = require('@hapi/boom')
+const processApplication = async (msg) => {
+  const reference = createReference()
+  await set({
+    reference,
+    type: 'VV001',
+    data: JSON.stringify(msg.body),
+    createdBy: 'admin',
+    updatedBy: 'admin',
+    updatedAt: new Date(),
+    createdAt: new Date()
+  })
+  const msgBody = msg.body
+  msgBody.applicationId = reference
+  sendMessage(msgBody, applicationResponseMsgType, applicationResponseQueue, { sessionId: msgBody.sessionId })
```

Comment:
Is this missing an `await`?

What happens if the message fails to send? I think it should be probably be wrapped in a `try...catch` and maybe have a `finally` where `sendEmail` is run. That would mean there would be an error on the frontend for the user but they should still get an email with the details.